client/state/account-recovery/reset/test/actions.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	fetchResetOptionsByLogin,
+	fetchResetOptionsByNameAndUrl,
+	fetchResetOptionsSuccess,
+	fetchResetOptionsError,
+} from '../actions';
+
+import {
+	ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
+	ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
+} from 'state/action-types';
+
+describe( '#fetchResetOptionsByLogin', () => {
+	it( 'should return ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST action with the user field', () => {
+		const user = 'foo';
+		const action = fetchResetOptionsByLogin( user );
+
+		assert.deepEqual( action, {
+			type: ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
+			userData: {
+				user,
+			},
+		} );
```

Comment:
you may find `expect()` from `chai` to be useful here.

```js
import { expect } from 'chai';

…

expect( action ).to.eql( { type: ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST, userData: { user } } );
```

It provides semantics that more closely map to written descriptions and sometimes it can provide more informative error messages than `assert` does.